### PR TITLE
[CA-2] Test PR

### PR DESCRIPTION
**Pull request by cyrxk-ui**
This PR removes the message boxes that were previously in these activities, since the returned result from either could be used in a message box in the invoking workflow.

### CODE REVIEW

File: HelloInput/Class1.cs [hunk 1]
```diff
@@ -5,7 +5,6 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using System.Windows.Forms;
 
 namespace HelloInput
```
Comment (psekaruipath) [inline HelloInput/Class1.cs]: It is highly recommended to create one file for one class. I see two classes in one file. Can you please break it down to two files? Also, name the files accordingly. For e.g. SquaredSumActivity.cs and SimpleSumActivity.cs

**Review by psekaruipath — CHANGES_REQUESTED** [review]
- Do you need to check in .suo file to this repo?
- I don't think you need to check in files under bin and obj folder? Can you add unwanted files/folders to .gitignore? https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/
- Another thing I would like you to figure out is how to add unit tests to these activities?

File: HelloInput/Class1.cs (hunk 1 above)
Comment (cyrxk-ui) [reply HelloInput/Class1.cs]: > * Do you need to check in .suo file to this repo?
> * I don't think you need to check in files under bin and obj folder? Can you add unwanted files/folders to .gitignore? https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/

Could you please have a look and see if my gitignore syntax is correct? I've tried every way I can think of and have not been able to exclude these files/folders

> * Another thing I would like you to figure out is how to add unit tests to these activities?

Added an executable project to the solution called UnitTests

Comment (cyrxk-ui) [reply HelloInput/Class1.cs]: > It is highly recommended to create one file for one class. I see two classes in one file. Can you please break it down to two files? Also, name the files accordingly. For e.g. SquaredSumActivity.cs and SimpleSumActivity.cs

Done

**Pull request merged**